### DISCUSSION TITLE
Link players/player features

### DISCUSSION THREAD

**Discussion by dannytrigo** [music-assistant/support#2052] [Feature requests and ideas]
I have a situation where I have a TV connected to an AV receiver with HDMI/eARC. They both support Chromecast and Airplay.

I start playback on the TV, and the audio comes from the receiver as expected. But when I try to control the volume, the TV can only step up/down the receiver volume by 0.5 at a time over HDMI. It can’t set it to a specific volume. So using the volume slider in MA I have to repeatedly slide it up/down, wait for it to return, repeat.

It would be great if I could link devices and configure that when I adjust TV volume, that command is actually sent to the receiver device instead (or to both)

Similarly, for a chromecast, it’s volume can’t be controlled due to the way it’s connected, and I’d like to forward the volume request to an Onkyo zone 2 receiver (media player entity in HA, but not a native player in MA).

I just discovered MA today and I’m absolutely loving it. Great project!

**Comment by OzGav** [comment]
This functionality is on the roadmap but not until after the 2.0 release

**Reply by dannytrigo** [reply]
That’s great to hear, thanks!

**Reply by OzGav** [reply]
In MA 2.4.0b23 and later mapping of player controls is available.
